django/db/backends/base/schema.py

```diff
@@ -200,6 +200,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = timezone.now()  # default for DateTimeField
+            internal_type = field.get_internal_type()
+            if internal_type == 'DateField':
+                default = default.date()
```

Comment:
The diff in MarkusH's above doesn't make auto_now(_add) timezone aware, it hardcodes them to be computed in UTC instead of using `settings.TIME_ZONE`, which doesn't make sense to me.